index.qmd

```diff
@@ -42,13 +53,13 @@ The diagram below depicts how some of the cloud-optimized formats discussed in t
 
 Notes:
 
-* Some data formats cover multiple data types, specifically:
-    * GeoJSON can be used for both vector and point data.
-    * HDF5 can be used for point data or data cubes (or both via groups).
-    * GeoParquet and FlatGeobuf can be used for vector data or point data.
-* LAS files are intended for 3D points, not 2D points (since COPC files are compressed LAS files, same goes for COPC files).
-* [TopoJSON](https://github.com/topojson/topojson) (an extension of GeoJSON that encodes topology) and [newline-delimited GeoJSON](https://stevage.github.io/ndgeojson/) are types of GeoJSON worth mentioning but not explicitly represented in the diagram.
-* GeoTIFF and GeoParquet are geospatial versions of the non-geospatial file formats TIFF and Parquet, respectively. FlatGeobuf builds upon the non-geospatial [flatbuffers](https://github.com/google/flatbuffers) serialization library (though flatbuffers is not a standalone file format)
+- Some data formats cover multiple data types, specifically:
+    - GeoJSON can be used for vector and point cloud data.
```

Comment:
Curious @wildintellect @kylebarron if you agree this should be "point" data or "point cloud" data?

Reply:
that's at least a little more clear

Reply:
point cloud is more clear @wildintellect ?

Reply:
I don't usually think of GeoJSON as supporting point cloud data, but I thought "point" would be included under the "vector" umbrella. So overall I don't necessarily see GeoJSON covering multiple geometry types.

Reply:
oh sorry, @kylebarron is right ... that line should say vector; point, lines, polygons

Point cloud should be mentioned somewhere else.